src/main/resources/templates/java/gradle_gradle/exercise/build.gradle

```diff
@@ -10,16 +11,14 @@ dependencies {
     implementation 'org.apache.commons:commons-lang3:3.12.0'
 }
 
-def assignmentSrcDir = "src"
-
 tasks.withType(JavaCompile) {
     options.encoding = 'UTF-8'
 }
 
 sourceSets {
     main {
         java {
-            srcDir assignmentSrcDir
+            srcDir "src"
```

Comment:
can we leave this out because it's the default?

Reply:
I tried this locally but when removing this source set, my local IDE (IntelliJ) does not recognize the `src`-Folder as a Source Files Directory anymore.

Reply:
This is because the default source directory in Gradle is `src/main/java` and not just `src`.
Resources would then be in `src/main/resources` (Currently the location has to be manually specified to be e.g. `resources`